deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -221,23 +221,63 @@ cephClusterSpec:
   #   # These labels can be passed as LabelSelector to Prometheus
   #   monitoring:
 
-  # resources:
-  #   # The requests and limits set here, allow the mgr pod to use half of one CPU core and 1 gigabyte of memory
-  #   mgr:
-  #     limits:
-  #       cpu: "500m"
-  #       memory: "1024Mi"
-  #     requests:
-  #       cpu: "500m"
-  #       memory: "1024Mi"
-  #   # The above example requests/limits can also be added to the other components
-  #   mon:
-  #   osd:
-  #   prepareosd:
-  #   mgr-sidecar:
-  #   crashcollector:
-  #   logcollector:
-  #   cleanup:
+  resources:
+    mgr:
+      limits:
+        cpu: "500m"
+        memory: "512Mi"
```

Let's allow the mgr to burst higher
```suggestion
        memory: "1Gi"
```